### PR TITLE
Support pruning for Instance Normalization

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**

### CODE REVIEW

File: nncf/dynamic_graph/graph.py [hunk 1]
```diff
@@ -172,6 +171,10 @@ class BaseModuleAttributes:
     def __init__(self, weight_requires_grad: bool):
         self.weight_requires_grad = weight_requires_grad
 
+    @staticmethod
+    def from_module(module):
+        return BaseModuleAttributes(module.weight.requires_grad)
```
Comment (vshampor) [inline nncf/dynamic_graph/graph.py:176]: Type hints will be necessary, ultimately, which will result in the explicit dependency of this class on PyTorch. @gadylshintr , do we need this dependency?

Comment (gadylshintr) [reply nncf/dynamic_graph/graph.py:176]: Currently I use the same `ModuleAttributes` classes for Pytorch and TF, hence I'd prefer to get rid of this dependency if possible.

Comment (AlexanderDokuchaev) [reply nncf/dynamic_graph/graph.py]: Added `_get_module_attributes` in nncf_network.py

**Review by vshampor — APPROVED** [review]

**Review by gadylshintr — APPROVED** [review]
